apps/api/src/bank-transactions-file/bank-transactions-file.module.ts

```diff
@@ -0,0 +1,44 @@
+import { Module } from '@nestjs/common'
+import { BankTransactionsFileService } from './bank-transactions-file.service'
+import { BankTransactionsFileController } from './bank-transactions-file.controller'
+import { PrismaService } from '../prisma/prisma.service'
+import { S3Service } from '../s3/s3.service'
+import { PersonService } from '../person/person.service'
+import { DonationsService } from '../donations/donations.service'
+import { VaultService } from '../vault/vault.service'
+import { CampaignService } from '../campaign/campaign.service'
+import { ConfigService } from '@nestjs/config'
+import { StripeModule } from '@golevelup/nestjs-stripe'
+import { Public } from 'nest-keycloak-connect'
+
+@Module({
+  imports: [
+    StripeModule.forRootAsync(StripeModule, {
+      inject: [ConfigService],
+      useFactory: async (config: ConfigService) => ({
+        apiKey: config.get('stripe.secretKey', ''),
+        webhookConfig: {
+          stripeWebhookSecret: config.get('stripe.webhookSecret', ''),
+          requestBodyProperty: 'body',
+          decorators: [
+            /**
+             * Avoid Keycloak @AuthGuard and @RoleGuard on Webhook controller
+             **/
+            Public(),
+          ],
+        },
+      }),
+    }),
```

Comment:
This is the second place in the app where we need this factory. Let's extract it in a single  factory service that can be reused instead of hard copy.

See https://docs.nestjs.com/fundamentals/custom-providers#factory-providers-usefactory